datasets/index.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: single
+author_profile: false
+title: Zarr Datasets
+sidebar:
+  title: "Content"
+  nav: sidebar
+---
+
+<font size="4">
+Zarr is <a href="https://zarr.dev/adopters/">adopted</a> by a variety of
+research groups, universities, organisations, and open-source communities.<br>
+
+Below is a list of a few examples of public datasets.<br><br>
+</font>
+
+| Dataset                                                                                                           | Hosting               | Size  |
+| ----------------------------------------------------------------------------------------------------------------- | --------------------- | ----- |
+| [CMIP6](https://console.cloud.google.com/marketplace/details/noaa-public/cmip6)                                   | Google Cloud          | 1 PB  |
+| [Image Data Resource (IDR)](https://idr.github.io/ome-ngff-samples/)                                              | EBI                   | 3 TB  |
+| [Microsoft Planetary Computer](https://planetarycomputer.microsoft.com/catalog?filter=zarr)                       | Microsoft Azure       |       |
+| [MUR Sea Surface Temperature](https://registry.opendata.aws/mur)                                                  | AWS Open Data Program | 54 TB |
+| [NASA POWER](https://registry.opendata.aws/nasa-power/)                                                           | AWS Open Data Program |       |
+| [National Water Model](https://discourse.pangeo.io/t/the-national-water-model-reanalysis-zarr-dataset-on-aws/1449)| Amazon AWS            | 5 TB  |
+| [NGFF](https://ngff.openmicroscopy.org/data/index.html)                                                           | Various providers     |       |
+| [webKnossos](https://zarr.webknossos.org/)                                                                        | scalableminds GmbH    | 70 TB |
+
+<font size="4">
+<br>If you think your dataset would be a useful example to add to the list,
+feel free to send a PR to the website <a href="https://github.com/zarr-developers/zarr-developers.github.io/">repository</a>.<br><br>
+
+Please ensure the dataset you're adding to the list is public, free to use,
+documented, represents Zarr's use case in a specific domain, and is stable/maintained. 
```

Comment:
Not sure I would understand "represents Zarr's use case in a specific domain. I would also expand on "is stable/maintained", e.g. "Please only include URLs that you expect to be stable and maintained for ..." (the next several years?)

Reply:
Thanks for the feedback!
Made some changes. Check here: https://zarr-developersgithubio--86.org.readthedocs.build/en/86/datasets/